lib/plaid/products/payment_initiation.rb

```diff
@@ -6,13 +6,15 @@ class PaymentInitiation < BaseProduct
     # name    - Recipient name.
     # iban    - Recipient IBAN.
     # address - Recipient address.
+    # bacs    - Recipient BACS (hash with "account" and "sort_code" keys)
     #
     # Returns a PaymentRecipientCreateResponse object.
-    def create_recipient(name, iban, address)
+    def create_recipient(name, iban, address, bacs)
```

Comment:
curious: is bacs a required field for creating recipients?

Reply:
no. I believe the client will be able to pass `null` in if they do not have it on hand.